spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -246,6 +246,27 @@ class Foo
     end
   end
 
+  context 'when ActiveSupport Inflector is defined', order: :defined do
+    before { require 'active_support/inflector' }
+
+    it 'registers an offense for a bad path when there is no custom acronym' do
+      expect_offense(<<-RUBY, 'pvp_class_foo_spec.rb')
+        describe PvPClass, 'foo' do; end
+        ^^^^^^^^^^^^^^^^^^^^^^^^ Spec path should end with `pv_p_class*foo*_spec.rb`.
+      RUBY
+    end
+
+    it 'does not register an offense when class name contains custom acronym' do
+      ActiveSupport::Inflector.inflections do |inflect|
+        inflect.acronym('PvP')
+      end
```

Comment:
I wonder if we couldn’t do
```rb
ensure
  ActiveSupport::Inflector.inflections.clear(:acronyms)
```
instead of having the `order :defined` on the context block.